.github/workflows/workflow.yml

```diff
@@ -7,91 +7,160 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - name: "1. Clone repository"
+        uses: actions/checkout@v2
 
-      - name: Set up Python 3.6
+      - name: "2. Set up Python 3.6"
         uses: actions/setup-python@v1
         with:
           python-version: 3.6
 
-      - name: Prepare machine
+      - name: "3. Update pip"
         run: |
           python -m pip install --upgrade pip
 
-      - uses: actions/cache@v1
+      - name: "4. Attache cache for pip"
```

Comment:
Not sure if 'Attache' is a typo or something I don't know about

Reply:
typo :)